blockchain/types/transaction.go

```diff
@@ -58,6 +59,13 @@ func deriveSigner(V *big.Int) Signer {
 	return LatestSignerForChainID(deriveChainId(V))
 }
 
+func ErrSender(err error) error {
+	return fmt.Errorf("sender: %s", err)
+}
+func ErrFeePayer(err error) error {
+	return fmt.Errorf("fee payer: %s", err)
+}
```

Comment:
Can we keep the original text "invalid sender"? I know that some inner errors say "invalid" again. But for the error messages to show up in Google search and docs, they should remain consistent.

Reply:
Sure, resolved it.